### PR TITLE
Do not log errors when a map was deleted out from under the game.

### PR DESCRIPTION
**Pull request by DanVanAtta**
----
 
1st commit - Handles the regression part of #1045 

Repro:
- download a map
- launch it
- open download map dialog, delete it
- launch game > notice error

After fix, when you launch the game, there is no map parse error, and the map name is set to "-"

-----

2nd commit - minor cleanup, but notably variable name cleanup

----

---
This change is [Reviewable](https://reviewable.io/reviews/triplea-game/triplea/1052)

### CODE REVIEW

**Comment by DanVanAtta** [comment]
This fixes a regression from 1.8.0.9

File: src/games/strategy/util/UrlStreams.java [hunk 1]
```diff
@@ -80,7 +78,6 @@ protected UrlStreams(Function<URL, URLConnection> connectionFactory) {
       connection.setUseCaches(false);
       return Optional.of(connection.getInputStream());
     } catch (IOException e) {
-      ClientLogger.logError("Failed to open a connection to: " + url, e);
```
Comment (RoiEXLab) [inline src/games/strategy/util/UrlStreams.java]: Why is this being removed?



Nothing else to say about this PR, looks good

**Comment by ron-murhammer** [comment]
@DanVanAtta Looks good to me as well and just the same question as RoiExLab.

File: src/games/strategy/util/UrlStreams.java (hunk 1 above)
Comment (DanVanAtta) [inline src/games/strategy/util/UrlStreams.java]: The answer is in the title: "Do not log errors when a map was deleted out from under the game."

The more interesting answer is that the game code determines that the map is missing by failing to read it. Implied in this is any client will need to log an error if they see an Optional.empty and if they cannot handle the situation.

**Pull request merged**